README.md

```diff
@@ -1,5 +1,8 @@
 # NFDI4Microbiota - Metadata Standards
 
+> [!TIP]
+> Find out about research data management, FAIR principles for microbiology and more on our [Knowledge base](https://knowledgebase.nfdi4microbiota.de)!
```

Comment:
Typing convention is "Knowledge Base"
Is there a quick fix to check across the whole github if the spelling is uniform?

Reply:
Ok thanks will change this. Yes locally but globally I"m not sure, maybe via the search function at the scale of the repository would already indicate to which extent the term was used.

Reply:
done in 5846c8f